src/server/server.ts

```diff
@@ -136,15 +139,44 @@ async function serveArticle(
 	res.end();
 }
 
+async function serveRichLinkDetails(
+	renderingRequest: RenderingRequest,
+	res: ExpressResponse,
+): Promise<void> {
+	const imageSalt = await getConfigValue('apis.img.salt');
+
+	if (imageSalt === undefined) {
+		throw new Error('Could not get image salt');
+	}
+
+	const docParser = JSDOM.fragment.bind(null);
+	const item = fromCapi({ docParser, salt: imageSalt })(renderingRequest);
+
+	if (
+		item.mainMedia.kind === OptionKind.Some &&
+		item.mainMedia.value.kind === MainMediaKind.Image
+	) {
+		res.set('image', item.mainMedia.value.image.src);
+	}
+
+	res.set('pillar', renderingRequest.content.pillarName);
+	res.status(200).end();
```

Comment:
Is your expectation that this returns an empty response with these two headers added? In PROD (after modifying the url to include `rendered-items`) I'm seeing a response that instead returns HTML _without_ these headers 🤔